cdap-client-tests/src/test/java/co/cask/cdap/client/ScheduleClientTestRun.java

```diff
@@ -91,5 +94,17 @@ public void testAll() throws Exception {
     scheduleClient.suspend(schedule);
     status = scheduleClient.getStatus(schedule);
     Assert.assertEquals("SUSPENDED", status);
+
+    List<ScheduledRuntime> scheduledRuntimes = scheduleClient.nextRuntimes(workflow);
+    Assert.assertEquals(1, scheduledRuntimes.size());
+    // simply assert that its scheduled for some time in the future (or scheduled for now, but hasn't quite
+    // executed yet
+    Assert.assertTrue(scheduledRuntimes.get(0).getTime() >= System.currentTimeMillis() + TimeUnit.MINUTES.toMillis(1));
```

Comment:
I think, this would fail if the test case is executed on 31st December between 11:59 PM and midnight :)